spec/file-icons-spec.coffee

```diff
@@ -21,3 +25,45 @@ describe 'FileIcons', ->
     FileIcons.resetService()
 
     expect(FileIcons.getService()).not.toBe(service)
+
+  
+  describe 'Class handling', ->
```

Comment:
What about moving this spec into `tree-view-spec.coffee`? Testing the interaction between `FileView` and the file icon service seems to belong more to that file. 💡